CODEOWNERS

```diff
@@ -21,12 +21,6 @@
 /datastreams                    @Datadog/data-streams-monitoring
 /internal/datastreams           @Datadog/data-streams-monitoring
 
-# telemetry
-/internal/telemetry             @DataDog/apm-go
-
-# linter rules
-.golangci.yml                   @DataDog/tracing-go @DataDog/profiling-go @DataDog/asm-go
-
```

Comment:
Note: I've removed the rules for telemetry and linter because they are covered by the default owner rule on top.